boot/config.js

```diff
@@ -234,11 +234,9 @@ boot.config({
 
     // THIRD PARTY: BOWER COMPONENTS
     // -----------------------------
-    promise: boot.bundle + 'stratus/bower_components/promise-polyfill/promise' + boot.suffix,
-
+    //promise: boot.bundle + 'stratus/bower_components/promise-polyfill/promise' + boot.suffix,
```

Comment:
Need a space in comments

Reply:
Also https://github.com/Sitetheory/stratus/blob/409618ce7bf6daf0a2fbc9196d02ae3b2048047b/source/external.js#L1 says there is a global $